scripts/mina-local-network/mina-local-network.sh

```diff
@@ -562,6 +572,11 @@ if ${VALUE_TRANSFERS}; then
     sleep 1
   done
 
+  echo "Set up zkapp account"
+  printf "\n"
+
+  QUERY=`${ZKAPP_EXE} create-zkapp-account --fee-payer-key ${FEE_PAYER_KEY_FILE} --nonce 0 --receiver-amount 10000 --zkapp-account-key ${ZKAPP_ACCOUNT_KEY_FILE} --graphql`
```

Comment:
Could you add some more zkapp transactions? like updating the state? It would be good to send such transactions periodically